examples/comparison.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2015, Ecole Polytechnique Federale de Lausanne, Blue Brain Project
+# All rights reserved.
+#
+# This file is part of NeuroM <https://github.com/BlueBrain/NeuroM>
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+#     1. Redistributions of source code must retain the above copyright
+#        notice, this list of conditions and the following disclaimer.
+#     2. Redistributions in binary form must reproduce the above copyright
+#        notice, this list of conditions and the following disclaimer in the
+#        documentation and/or other materials provided with the distribution.
+#     3. Neither the name of the copyright holder nor the names of
+#        its contributors may be used to endorse or promote products
+#        derived from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY
+# DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+'''Module for the comparison of the morphometrics between two sets of trees.'''
+
+import numpy as np
+from neurom.core import iter_neurites
+
+
+def get_mod_features(object1, object2, flist, collect_all, mod):
+    '''Computes features from module mod'''
+    for feat in flist:
+
+        feature_pop = np.array([l for l in iter_neurites(object1, getattr(mod, feat))])
+        feature_neu = np.array([l for l in iter_neurites(object2, getattr(mod, feat))])
+
+        # Standardization of data: (data - mean(data))/ std(data)
+        m = np.mean(feature_pop)
+        st = np.std(feature_pop)
+
+        collect_all.append([(feature_pop - m) / st, (feature_neu - m) / st])
+
+    return collect_all
+
+
+def get_features(object1, object2, tri_list=('meander_angle',),
```

Comment:
I don't think it is a good idea to pass the features as strings and then try to pull them from a module. Better to make a map, for example,

```python
feature_map = {'segment_length': segments.length, 'section_length': sections.length, ...}
```

and so on. Less implicit stuff, and less scope for difficult to debug runtime errors.